### ISSUE TITLE
sbt bgRun crashes with NoSuchMethodException: play.core.server.DevServerStart.main

### ISSUE THREAD

**Issue by giannoug** [playframework/playframework#9505]
### Play Version (2.5.x / etc)

2.7.3

### API (Scala / Java / Neither / Both)

2.12

### Operating System (Ubuntu 15.10 / MacOS 10.10 / Windows 10)

Windows 10 Pro (Version 1903)

### JDK (Oracle 1.8.0_72, OpenJDK 1.8.x, Azul Zing)

```
java version "1.8.0_211"
Java(TM) SE Runtime Environment (build 1.8.0_211-b12)
Java HotSpot(TM) 64-Bit Server VM (build 25.211-b12, mixed mode)
```

### Expected Behavior

Please describe the expected behavior of the issue, starting from the first action.

1. First open an sbt shell
2. Use bgRun to make Play run in background mode

### Actual Behavior

Application startup crashes with the following output:
```
[info] Running play.core.server.DevServerStart 
[error] (run-main-1) java.lang.NoSuchMethodException: play.core.server.DevServerStart.main([Ljava.lang.String;)
[error] java.lang.NoSuchMethodException: play.core.server.DevServerStart.main([Ljava.lang.String;)
[error]        at java.lang.Class.getMethod(Unknown Source)
```

### Reproducible Test Case

1. sbt new playframework/play-scala-seed.g8
2. cd play-scala-seed
3. sbt
4. bgRun

**Comment by renatocaval** [comment]
This is probably happening because `brRun` is not using the same classloader as Play's `run` uses. 

@marcospereira, do you have some more hints on this?

**Comment by renatocaval** [comment]
@giannoug, I looked it further and here is what is happening. 

When you run `bgRun` you are relying solely on sbt and therefore it tries to find a `main` method which is not present in `play.core.server.DevServerStart`.

Play redefines the `run` task in other to support the reloading. Play's run task know about `play.core.server.DevServerStart` and how to start it. 

So, basically you can't use `bgRun` with Play unless you redefine it. The best would be to define a similar task that relies on Play's default run task but puts the process in the background. 

If you want you can send a PR adding support to `bgRun` in Play. Feel free to re-open this issue if you decide so.

**Comment by mkurz** [comment]
`bgRun` support for Play should be soon become a reality:
* Implemented in #11441
* Depends on https://github.com/sbt/sbt/pull/7147
